File: OutArray/out.java
class out {
    public static void main(String[] args) {
        String[] input = {"This", "is", "an", "array"};
        int max = 4;
        for (int x=0; x<=max; x++) {
            System.out.printf("Index: %d // Value: %s\n", x, input[x]);
        }
        System.out.println("Goodbye, I'm Java");
    }
}

File: OutArray/out.pl
#!/usr/bin/perl
use strict;
my @input = ("This", "is", "an", "array");
my $max = 4;
for (my $x=0; $x<=$max; $x++) {
    printf("Index: %d // Value: %s\n", $x, $input[$x]);
}
printf "Goodbye, I'm Perl!\n";
